pages/linux/tomb.md

```diff
@@ -0,0 +1,31 @@
+# tomb
+
+> Manage encrypted storage folders that can be safely transported and hidden in a filesystem.
+
+- Create a new tomb with an initial size of 100MB:
+
+`tomb dig -s 100 {{encrypted_folder.tomb}}`
+
+- Create a new key file that can be used to lock a tomb, and set its password:
```

Comment:
I like the style we followed in `mongo` - "Connect to a database with a given username; user will be prompted for password".